### PR TITLE
44 todo list

### PR DESCRIPTION
**Pull request by shayaansultan**
Add basic tasks list functionality for each employee

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AddTaskCommand.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+     * @return A {@code CommandResult} representing the result of the command
+     *         execution.
+     * @throws CommandException If an error occurs during command execution.
+     */
+    private CommandResult addTask(Model model) throws CommandException {
+        List<Employee> lastShownList = model.getFilteredEmployeeList();
+        Employee employee = null;
+
+        for (Employee e : lastShownList) {
+            assert uid != null;
```
Comment (hjungwoo01) [inline src/main/java/seedu/address/logic/commands/AddTaskCommand.java:52]: Nice use of assertions

**Review by PyromancerBoom — COMMENTED** [review]
Looks good to me

File: src/main/java/seedu/address/model/task/Task.java [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        }
+        return Description.isValidDescription(taskStringArray[1]);
+    }
+
+    /**
+     * Parses a task string into a Task object.
+     */
+    public static Task parseTaskString(String taskString) {
+        String[] taskStringArray = taskString.split(" ", 2);
+        boolean isDone = taskStringArray[0].equals("[X]");
```
Comment (hjungwoo01) [inline src/main/java/seedu/address/model/task/Task.java:111]: This works but would it be better if the Task object contained the isDone variable?

**Review by hjungwoo01 — APPROVED** [review]
LGTM

**Pull request merged**